corehq/apps/hqwebapp/static/hqwebapp/js/components/bootstrap3/feedback.js

```diff
@@ -74,4 +76,17 @@ hqDefine('hqwebapp/js/components/bootstrap3/feedback', [
         },
         template: '<div data-bind="template: { name: \'ko-feedback-template\' }"></div>',
     };
+
+    ko.components.register('feedback', component);
+
+    $(function () {
+        _.each($('feedback'), function (el) {
+            var $el = $(el);
+            if (!($el.data('apply-bindings') === false)) {
```

Comment:
nit: is there a reason the `!==` operator was not used here?
```suggestion
            if ($el.data('apply-bindings') !== false) {
```

Reply:
nope, that's a good update